perf/rt_tests.py.data/rt_tests.yaml

```diff
@@ -0,0 +1,8 @@
+setup:
+    general: !mux
+         default:
+             test_to_run: signaltest
+             args: '-t 10 -l 100000 '
+         cyclictest: !mux
+             test_to_run: cyclictest
+             args: '-t 10 -l 100000 '
```

Comment:
Could you please also add configuration for the other tests so we have all of them covered? There is an outstanding PR for pi_test https://github.com/avocado-framework/avocado-misc-tests/pull/71 so having all tests defined could help prevent the duplicity. I'd recommend something like:

```
pi_test:
    test_to_run: pi_stress
    args: "--duration=300"
```

for the `pi_test` and other suitable defaults for the rest of the tests.